packages/wrangler/src/config/validation.ts

```diff
@@ -128,10 +128,17 @@ export function normalizeAndValidateConfig(
 		);
 	}
 
+	const isDispatchNamespace = typeof args["dispatch-namespace"] === "string";
+
 	const topLevelEnv = normalizeAndValidateEnvironment(
 		diagnostics,
 		configPath,
-		rawConfig
+		rawConfig,
+		undefined,
+		undefined,
+		undefined,
+		undefined,
+		isDispatchNamespace
```

Comment:
I know technically it's the same, but should we add support for `isDispatchNamespace` in the first fn signature, instead of passing all these `undefined` params? Would make things much cleaner IMHO, which I assume was the reason behind having this particular fn signature to begin with

Reply:
Yeah we could move it lower down

Reply:
This actually isn't needed if we do the normal script name validation

Reply:
> but there's a sig with fully required args which we couldn't put it in without it going to the end

Reply:
Haha yes very good point, moved.